release_tools/release.sh

```diff
@@ -0,0 +1,392 @@
+version=1.2.17
+previous_version=1.2.16
+latest_fedora=28
+latest_rhel=7
+build_for_rhel=('rhel-7.5')
+version_major_minor="${version%.*}"
+
+JENKINS_SITE='https://jenkins.open-scap.org'
+GITHUB_ROOT='https://github.com/OpenSCAP/openscap'
+
+test -f .env && . .env
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+OSCAP_REPO_ROOT="$(cd "$SCRIPT_DIR" && cd ../.. && pwd)"
+LIBRARY_STASH="/tmp/library-stash"
+BUILDDIR="$OSCAP_REPO_ROOT"
+
+
+die()
+{
+    echo "$1"
+    exit 1
+}
+
+
+clean_repository()
+{
+    make distclean
+    git clean -f
+}
+
+
+clean_repository_aggressively()
+{
+    git reset --hard
+    git clean -x -f
```

Comment:
I personally add -d to clean directories as well, eg. when I switched from a master branch. I don't know if it can cause problems with release if there is some unrelated directory, but I am careful